src/components/Button/Button.tsx

```diff
@@ -135,3 +135,4 @@ const ButtonComponent: ForwardRefRenderFunction<HTMLButtonElement | null, Button
 };
 
 export const Button = forwardRef(ButtonComponent);
+Button.displayName = 'Button';
```

Comment:
Why `AddBlockButton` and `Button` don't get the prefix?

Reply:
Added manually, forgotten to add the prefix.